src/java/com/unifina/security/SessionToken.java

```diff
@@ -0,0 +1,32 @@
+package com.unifina.security;
+
+import com.unifina.domain.security.SecUser;
+import org.apache.commons.lang.RandomStringUtils;
+import org.joda.time.DateTime;
+
+import java.io.*;
+import java.util.Base64;
+
+public class SessionToken{
+	private String token;
+	private SecUser user;
+	private DateTime expiration;
```

Comment:
We tend to use plain old `java.util.Date` objects as they automatically play nice with Grails - might be worth considering here, if no complex datetime manipulation is needed